Tasks/GitHubReleaseV0/task.json

```diff
@@ -187,6 +187,28 @@
             "required": false,
             "visibleRule": "action = create || action = edit",
             "helpMarkDown": "If set to `true`, a list of changes (commits and issues) between this and the last published release will be generated and appended to the release notes."
+        },
```

Comment:
change either patch or minor version of the task.

Reply:
Version 0.155 is not live yet. Hence didn't change the version.